recipes/scikit-fdiff/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikit-fdiff" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-fdiff-{{ version }}.tar.gz
+  sha256: 05d39e607c553cbc3b3ec328b84a6f2665291cd095ef94ae8479819ad4fbe3ec
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - numba
+    - scipy
+    - sympy <1.6
+    - streamz
+    - more-itertools
+    - xarray
+    - dask-core
+    - pyyaml
+    - joblib
+    - path
```

Comment:
It looks like this should be `path.py` rather than `path` according to the upstream `setup.cfg`. I'm not sure if you'll need quotation marks around it due to the dot in the name. Could be fine without.

Reply:
I shall now try with quotation marks. Previously it did not work without them.